gen/calc.py

```diff
@@ -1450,8 +1450,8 @@ def calculate_fault_domain_detect_windows_contents(fault_domain_detect_windows_f
         'diagnostics_bundles_dir': '/var/lib/dcos/dcos-diagnostics/diag-bundles',
         'fault_domain_detect_filename': 'genconf/fault-domain-detect',
         'fault_domain_detect_contents': calculate_fault_domain_detect_contents,
-        'fault_domain_detect_windows': 'genconf/fault-domain-detect.ps1',
-        'fault_domain_detect_windows_contents': calculate_fault_domain_detect_contents,
+        'fault_domain_detect_windows_filename': 'genconf/fault-domain-detect.ps1',
```

Comment:
Where is this file?

Reply:
It doesn't need to exist, since `calculate_fault_domain_detect_contents` checks whether it exists and proceeds anyway, using a default. The problem with the initial commit is that the returned empty string does not output the correct format for a fault domain.